src/test/unit_tests/test_s3_ops.js

```diff
@@ -5,16 +5,15 @@ const _ = require('lodash');
 // setup coretest first to prepare the env
 const coretest = require('./coretest');
 coretest.setup({ pools_to_create: coretest.POOL_LIST });
-
+const config = require('../../../config');
 const AWS = require('aws-sdk');
 const http = require('http');
 const mocha = require('mocha');
 const assert = require('assert');
 const querystring = require('querystring');
-const SKIP_TEST = !process.env.AWS_ACCESS_KEY_ID || !process.env.AWS_SECRET_ACCESS_KEY;
 
-const AWS_TARGET_BUCKET = 's3-ops-test-bucket';
-const AWS_SOURCE_BUCKET = 's3-ops-source';
+// Uses the noobaa endpoint to create buckets for namespace and namespace cache bucket testing.
+const USE_LOCAL_ENDPOINT = process.env.USE_LOCAL_ENDPOINT === 'true';
```

Comment:
Hmm, didn't we mean the default to be the opposite?

Reply:
Well, using noobaa bucket as default will make the tests on travis not dependent on an external resource. 

If we make the external bucket as default for the s3 unit tests, the s3 test needs to be updated so that a unique external bucket is created for each pull request (does not happen at the moment).

If external endpoint should be the default, I can do that in the next pull request.
I need to create an external bucket with the commit id to ensure uniqueness

Reply:
by default I meant that if we do not set USE_LOCAL_ENDPOINT then it should run the default mode.
So is it going to be looking for external endpoint if I do * * * not * * * set USE_LOCAL_ENDPOINT?

Reply:
I am trying to say that I think we want to reverse this env meaning to:
```
const USE_REMOTE_ENDPOINT = process.env.USE_REMOTE_ENDPOINT === 'true';
```

Reply:
Ok, sorry, I misunderstood, I will do that.